assets/docker/mercury/docker-compose.yml

```diff
@@ -100,15 +110,27 @@ services:
       AUTH0_MERCURY_API_CLIENT_SECRET: ${AUTH0_MERCURY_API_CLIENT_SECRET}
       SENDGRID_API_KEY: fake_key
       LD_SDK_KEY: ${LD_SDK_KEY}
+      GX_SCHEDULER_LAMBDA_ARN: ${GX_SCHEDULER_LAMBDA_ARN}
+      GX_SCHEDULER_EXECUTION_ROLE_ARN: ${GX_SCHEDULER_EXECUTION_ROLE_ARN}
+      GX_SCHEDULER_EVENTBRIDGE_DLQ_ARN: ${GX_SCHEDULER_EVENTBRIDGE_DLQ_ARN}
+      GX_SCHEDULER_EVENTBRIDGE_REGION: ${LOCALSTACK_AWS_DEFAULT_REGION}
+      AWS_ENDPOINT_URL: ${LOCALSTACK_AWS_ENDPOINT_URL}
+      AWS_DEFAULT_REGION: ${LOCALSTACK_AWS_DEFAULT_REGION}
+      AWS_SECRET_ACCESS_KEY: ${LOCALSTACK_AWS_SECRET_ACCESS_KEY}
+      AWS_ACCESS_KEY_ID: ${LOCALSTACK_AWS_ACCESS_KEY_ID}
     depends_on:
       mercury-service-api:
         condition: service_started
       db-provisioner:
         condition: service_completed_successfully
+      localstack:
+        condition: service_healthy
     entrypoint:
       - /bin/bash
       - -c
       - |
+        cd /app/services/ge_cloud
+        ./scripts/localstack_init.sh
```

Comment:
I think you can remove the 2 alembic lines in this entrypoint since they are about setting the db up and the v0 mercury service already does that.